gap/pcpgrp/centcon.gi

```diff
@@ -165,7 +165,7 @@ CentralizerPcpGroup := function( G, g )
 
     # check
     if ForAny( g, x -> not x in G ) then
-        Error("elements must be contained in group");
+        TryNextMethod();
```

Comment:
I haven't checked yet, but: is this actually a requirement for `CentralizerBySeries` to work correctly? Or is perhaps the test above wrong and instead the check is supposed to check that the `g` are elements of the parent of G / the full group of the underlying collector?

Reply:
Unfortunately, it seems like this is really a requirement. In both the nilpotent and in the general case I get incorrect results for the centralizer.
```
gap> G := SmallGroup( 16, 8 );; # QD16, Nilpotent
gap> U := Subgroup( G, [ G.2*G.3*G.4 ] );;
gap> g := G.1*G.2*G.3*G.4;;
gap> Centralizer( U, g );
Group([  ])
gap> iso := IsomorphismPcpGroup( G );;
gap> G := Image( iso, G );;U := Image( iso, U );;g := Image( iso, g );;
gap> CentralizerPcpGroup( U, g );
Pcp-group with orders [ 2 ]
gap> CentralizerNilpotentPcpGroup( U, g );
Pcp-group with orders [ 2 ]
```